<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selectscene</class>
 <widget class="QMainWindow" name="Selectscene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>198</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="cbx1"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbx2"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btn">
       <property name="text">
        <string>确认选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="separator"/>
    <addaction name="quit"/>
    <addaction name="return_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="return_2">
   <property name="text">
    <string>返回</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
